submissions/description/server_side_injection/remote_code_execution_rce/recommendations.md

```diff
@@ -0,0 +1,5 @@
+# Recommendation(s)
+
+RCE attacks are challenging to prevent because the chain of execution to effect entry can vary widely. The key to minimizing the number of vulnerabilities in the environment is to patch and update all software in a consistent and repeatable update cycle.
```

Comment:
I'd delete this line and expand on the second one to state that keeping consistent and to date with patching is important if the vulnerability is in a 3rd party component.